pkgs/development/compilers/gcc/common/libgcc-buildstuff.nix

```diff
@@ -47,11 +47,14 @@ in
 # https://www.openwall.com/lists/musl/2022/11/09/3
 #
 # 'parsed.cpu.family' won't be correct for every platform.
-+
-  lib.optionalString
-    (
-      stdenv.targetPlatform.isLoongArch64 || stdenv.targetPlatform.isS390 || stdenv.targetPlatform.isAlpha
-    )
-    ''
-      touch libgcc/config/${stdenv.targetPlatform.parsed.cpu.family}/crt{i,n}.S
-    ''
++ (lib.optionalString
+  (
+    stdenv.targetPlatform.isLoongArch64 || stdenv.targetPlatform.isS390 || stdenv.targetPlatform.isAlpha
+  )
+  ''
+    touch libgcc/config/${stdenv.targetPlatform.parsed.cpu.family}/crt{i,n}.S
+  ''
+)
++ lib.optionalString (stdenv.targetPlatform.isPower && !stdenv.targetPlatform.isPower64) ''
+  touch libgcc/config/rs6000/crt{i,n}.S
```

Comment:
Just checking: rs6000 isn't a CPU family like the above case, is it?

Reply:
yes, for powerpc it would be just "power", so I can't reuse the thing above.